common/content/commandline.js

```diff
@@ -19,8 +19,10 @@ const CommandLine = Module("commandline", {
 
         this._callbacks = {};
 
-        storage.newArray("history-search", { store: true, privateData: true });
-        storage.newArray("history-command", { store: true, privateData: true });
+        ["search", "command"].forEach(function (mode) {
+            storage.newArray(liberator.storeName(mode, liberator.isPrivateWindow()),
+                { store: true, privateData: true });
```

Comment:
Thanks @caisui, I think you're right. The _'store'_ property should be _'false'_ in private mode and the _privateData_ property is redundant. The pull request has been updated accordingly.